ai/voice completion creator/68ca127a-2a6a-46c4-aaea-3651cd5511e9.json

```diff
@@ -185,10 +185,17 @@
     "ask": null,
     "question": null,
     "answer": null,
-    "openAI": null
+    "openAI": null,
+    "azure": null,
+    "ChatGPT": null,
+    "chat": null,
+    "gpt": null,
+    "open": null,
```

Comment:
I don't think "open" is a valid keyword for this prototype it is rather a misleading one.
Basically if we want this prototype to appear in the search when the user searches for "openai" or "open ai" we don't need to include into the keywords "openai", "open" and "ai" since searching for "open ai" will also find the node as both of the search terms will match the "openai" keywork partially.
Not blocker just a non recommended practice IMHO.